app/views/notes/_form.html.erb

```diff
@@ -32,18 +33,22 @@
 
                     <p>
                     <div class="form-group">
-                        <%= f.label :visit_date %> <%= f.date_field :visit_date, value: Date.today, class: "form-control" %>
+                        <% if defined?(@note) %>
```

Comment:
Instead of an if/else block with repeated code, can you just use a ternary operator?
`<%= f.label :visit_date %> <%= f.date_field :visit_date, value: defined?(@note) ? @note.visit_date : Date.today, class: "form-control" %>`